lib/controller.js

```diff
@@ -686,32 +686,11 @@ controller.connectToNetwork = function(opts) {
         return reject('Invalid credentials: must set a password with the -p or --password option.');
       }
 
-      if (security && securityOptions.indexOf(security) < 0) {
-        return reject(`"${security}" is not a valid security option. Please choose on of the following: ${securityOptions.join(', ')}`);
-      }
-
-      if (security === 'wep') {
-        // WEP passphrases can be 10, 26, or 58 hexadecimal digits long.
-        // Match for hexadecimal characters:
-        if (isNaN(`0x${password}`)) {
-          return reject('Invalid passphrase: WEP keys must consist of hexadecimal digits, i.e. 0 through 9 and "a" through "f".');
-        }
-        // Then test for length:
-        const length = password.length;
-        if (length !== 10 || length !== 26 || length !== 58) {
-          return reject('Invalid passphrase: WEP keys must be 10, 26, or 58 digits long for 64-, 128-, and 256-bit WEP.');
-        }
-      }
-
-      if (security === 'psk' || security === 'psk2') {
-        // WPA/WPA2-PSK passphrases can be 8-63 ASCII characters, or 64 hexadecimal digits.
-        // Match ASCII codes for all 127 printable ASCII characters:
-        const asciiRegex = /^[\x00-\x7F]{8,63}$/;
-        // Match exactly 64 hexadecimal digits:
-        const hexTest = !isNaN(`0x${password}`) && password.length === 64;
-        // Reject if both tests fail.
-        if (!asciiRegex.test(password) && !hexTest) {
-          return reject('Invalid passphrase: WPA/WPA2-PSK passkeys must be 8-63 ASCII characters, or 64 hexadecimal digits.');
+      if (security) {
+        if (securityOptions.indexOf(security) < 0) {
+          return reject(`"${security}" is not a valid security option. Please choose on of the following: ${securityOptions.join(', ')}`);
```

Comment:
- typo: "Please choose on of" => "Please choose one of"
- While you're in there, make the condition: `securityOptions.indexOf(security) === -1`, just to be as clear as possible (I wasn't going to mention this, but then I saw the typo and figured "why not?")